java/src/main/java/com/powsybl/dataframe/network/adders/NetworkUtils.java

```diff
@@ -28,6 +26,30 @@ public static VoltageLevel getVoltageLevelOrThrow(Network network, String id) {
         return voltageLevel;
     }
 
+    public static VoltageLevel getVoltageLevelOrThrowWithBusOrBusbarSectionId(Network network, int row, StringSeries voltageLevels, StringSeries busOrBusbarSections) {
+        VoltageLevel voltageLevel;
```

Comment:
not important you delete these and return directly the voltage level in the if sections